src/nsrt_learning/sampler_learning.py

```diff
@@ -147,11 +158,19 @@ def _learn_neural_sampler(datastores: List[Datastore], nsrt_name: str,
     print("Fitting regressor...")
     X_regressor: List[List[Array]] = []
     Y_regressor = []
-    for state, sub, option in positive_data:  # don't use negative data!
+    for state, sub, option, goal in positive_data:  # don't use negative data!
         # input is state features
         X_regressor.append([np.array(1.0)])  # start with bias term
         for var in variables:
             X_regressor[-1].extend(state[sub[var]])
+        # See comment above.
```

Comment:
There are many comments above. Be more clear, e.g:
"Above, we made the assumption that (blah), which must also be used when assembling data for the regressor."

Reply:
fixed!